docs/release-notes.txt

```diff
@@ -8,6 +8,52 @@
  Version 0.14
 **************
 
+Version 0.14.6
+==============
+
+**Release Date:** April 1, 2021
+
+**New Features**
+
+-  API: Add a new endpoint to delete experiments. This endpoint is only
+   enabled for admin users and deletes all resources associated with an
+   experiment. This includes checkpoint storage, Tensorboards, trial
+   logs from all backends and metadata such as history and metrics,
+   stored in PostgreSQL.
+
+-  API: Add a new endpoint to fetch aggregated historical resource
+   allocation information.
+
+-  CLI: Add new commands ``det resources raw`` and ``det resources
+   aggregated`` to access resource allocation information.
+
+-  PyTorch Lightning: Add an adapter to support ``LightningModule`` from
+   PyTorch Lightning in the PyTorchTrial API. :ref:`Read more about this
+   in the documentation here <lightning-adapter>`.
+
+-  WebUI: Experiment visualizations attempt to pick the most relevant
+   hyperparameters to visualize by default, using HP importance if
+   available.
+
+-  WebUI: Add support for showing categorical hyperparameters in HP
```

Comment:
non-blocking: For less redundancy, maybe "Improve the hyperparameter scatter plot and heat map visualizations by adding support for showing categorical hyperparameters."? also i'd move this to improvements.